lintro/utils/output/manager.py

```
@@ -56,27 +60,93 @@ def __init__(
+            except FileExistsError:
+                suffix += 1
 
     def _create_run_dir(self) -> Path:
         """Create a new timestamped run directory.
 
         Returns:
             Path: Path to the created run directory.
         """
-        timestamp: str = datetime.datetime.now().strftime(DEFAULT_TIMESTAMP_FORMAT)
-        run_dir: Path = self.base_dir / f"{DEFAULT_RUN_PREFIX}{timestamp}"
+        timestamp: str = datetime.datetime.now().strftime(
```

Ruff (0.15.10)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)